experimental/datacommons-js/README.md

```diff
@@ -12,11 +12,35 @@ Include datacommons.js and styles in your html `<head>...</head>` tag
 <script src="https://datacommons.org/datacommons.js"></script>
 ```
 
+For testing on autopush, replace the above script tag with
```

Comment:
can remove "www." from the two stylesheet links above

Reply:
Removed